packages/services/src/kernel/kernel.ts

```diff
@@ -537,6 +594,13 @@ export interface IKernelConnection extends IObservableDisposable {
       'clientId' | 'username' | 'handleComms'
     >
   ): IKernelConnection;
+
+  supportsSubshells(): boolean;
```

Comment:
Similar to the comment above: a small docstring here would be useful for consistency.